public/js/twostep.js

```diff
@@ -0,0 +1,194 @@
+// Auto
+  $(function() {
+
+    // Check for on keypress
+    $("input").on("keydown", function(event) {
+
+      var self = $(this);
+
+      // Keyboard Controls
+      var controls = [8, 16, 18, 17, 20, 35, 36, 37, 38, 39, 40, 45, 46, 9, 91, 93, 224, 13, 127, 27, 32];
+
+      // Special Chars
+      var specialChars = [43, 61, 186, 187, 188, 189, 190, 191, 192, 219, 220, 221, 222];
+
+      // Numbers
+      var numbers = [48, 49, 50, 51, 52, 53, 54, 55, 56, 57];
+
+      var preCombined = controls.concat(numbers);
+      var combined = preCombined;
+
+      // Allow Letter
+      for (var i = 65; i <= 90; i++) {
```

Comment:
Replace that with a range,  and concat it with the combined array later. http://php.net/manual/en/function.range.php